README.md

```diff
@@ -1,15 +1,21 @@
-# A cron-like time-based job scheduler for Dart
+<p align="center" width="100%">
+  <img width="50%" src="https://github.com/mbfakourii/cron/assets/20955005/6a8a8064-7167-462a-a485-36565641a71f" alt="logo"/>
```

Comment:
Please let's remove this part (and also the image) for now. While this is a cool logo, I don't think neither this package, or its unix namesake needs a logo - it is just a small although important utility after all.

Reply:
Sure

Reply:
Done